base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py

```diff
@@ -12,11 +12,18 @@
 table = widget.getTable()
 row = 1
 color = ColorFontUtil.getColorFromRGB(240,245,245)
+
 i = 0
 for i in range(20):
-    reading = PVUtil.getString(pvs[i]) 
+    reading = PVUtil.getString(pvs[i])
+    temperature = PVUtil.getString(pvs[i+20])
+    drift = PVUtil.getString(pvs[i+40])
```

Comment:
2nd magic number. Is this `2*offset_between_temperature_and_drift_pv_indexes` or something else? I can't know without examining the entire code, reducing overall readability

Reply:
Removed